producers/pipsafety/BUILD

```diff
@@ -0,0 +1,34 @@
+subinclude("@pleasings//docker")
+
+go_binary(
+    name = "python_pipsafety",
```

Comment:
Same as the other diff. It's nice if the main build target matches the directory name. So using `name = "pipsafety"` would be nice here.